tests/integration/yarn_classic/conftest.py

```
@@ -0,0 +1,77 @@
+@pytest.fixture(scope="session")
+def nexus_collision_tarballs() -> None:
+    """Upload two tarballs with identical filenames to different Nexus paths.
+
+    Both are served as ``/a/example.tgz`` and ``/b/example.tgz`` from a hosted repository, so yarn's
+    offline-mirror naming produces a collision.
+    """
+    # Testing this without the nexus instance we already have would require yet another git repo to
+    # host the packages
+    assert os.getenv("HERMETO_TEST_LOCAL_NEXUS") == "1"
+
+    requests.post(
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)